source/_posts/2024-05-01-release-20245.markdown

```diff
@@ -51,8 +51,7 @@ of you requested since the last update. But the ability to change the names of
 the devices shown on the energy dashboard is a close second!
 
 Oh! We've held our [State of the Open Home 2024 live stream](https://www.youtube.com/watch?v=oa__fLArsFk)!
-In case you've missed it, I would definitly recommend on watching it back.
-If you’ve missed it, I recommend watching it back. There are tons of cool things
+In case you've missed it, I would definitly recommend on watching it back.  There are tons of cool things
```

Comment:
Typo in `definitely` and double space removal.

```suggestion
In case you've missed it, I would definitely recommend watching it back. There are tons of cool things
```